internal/pkg/deploy/env.go

```diff
@@ -13,7 +13,7 @@ const (
 	// LegacyEnvTemplateVersion is the version associated with the environment template before we started versioning.
 	LegacyEnvTemplateVersion = "v0.0.0"
 	// LatestEnvTemplateVersion is the latest version number available for environment templates.
-	LatestEnvTemplateVersion = "v1.6.1"
+	LatestEnvTemplateVersion = "v1.6.2"
```

Comment:
nit: Technically it's a feature and not a bug fix :P 
```suggestion
	LatestEnvTemplateVersion = "v1.7.0"
```